scripts/create_perf_json.py

```diff
@@ -1861,11 +1899,20 @@ def main():
                     default=0,
                     dest='verbose',
                     help='Additional output when running.')
+    ap.add_argument('--event_only',
+                    '-e',
+                    type=bool,
+                    action=argparse.BooleanOptionalAction,
+                    default=False,
+                    help='Update event json files only.')
     args = ap.parse_args()
 
     global _verbose
     _verbose = args.verbose
 
+    global _event_only
+    _event_only = args.event_only
```

Comment:
I'm not sure about using a global variable. It would help clarify `Model` if distinct actions for events and metrics were split into their own methods. What do you think about `Model.to_perf_json()` calling two helpers such as `create_perf_events()` and `create_perf_metrics()` depending on the command line argument?

Reply:
I commented above, I think the change is fine. I'm not sure on the motivation as I tend to always generate both events and metrics, so I can compare with what is in the Linux tree. If I only generate one then I'd expect to get differences. I also want the events and metrics to be in sync so that I don't get metrics referencing events that don't exist.